### PR TITLE
[GPU] IsFinite, IsInf, IsNaN operations

### PR DESCRIPTION
**Pull request by mhnap-lohika**
### Details:
 - *Implementation of new operations in GPU plugin: IsFinite, IsInf, IsNan*

### Tickets:
 - *94742*, *94754*, *94722*

### CODE REVIEW

**Review by wilson-seok — APPROVED** [review]

**Comment by mhnap-lohika** [comment]
Also, here are the conformance test OpImplCheck results.
image

File: src/plugins/intel_gpu/src/graph/eltwise.cpp [hunk 1]
```diff
@@ -408,6 +423,14 @@ void eltwise_inst::check_inputs_count(eltwise_node const& node) {
                     node.id(),
                     "Invalid eltwise inputs number (should be equal to 2). Actual: " + std::to_string(inputs_number));
             break;
+        case eltwise_mode::is_finite:
+        case eltwise_mode::is_inf:
+        case eltwise_mode::is_nan:
+            if (inputs_number != 1)
+                CLDNN_ERROR_MESSAGE(
```
Comment (yeonbok) [inline src/plugins/intel_gpu/src/graph/eltwise.cpp:430]: Please use OPENVINO_ASSERT.

**Review by yeonbok — APPROVED** [review]

**Comment by yeonbok** [comment]
LGTM except minor comment.

File: src/plugins/intel_gpu/src/graph/eltwise.cpp (hunk 1 above)
Comment (mhnap-lohika) [reply src/plugins/intel_gpu/src/graph/eltwise.cpp:430]: Used as above to be consistent. But if needed, I can change both places to use OPENVINO_ASSERT.

Comment (yeonbok) [reply src/plugins/intel_gpu/src/graph/eltwise.cpp:430]: Yes those are remaining legacy and we are replacing with OPENVINO_ASSERT for newly added codes.

Comment (mhnap-lohika) [reply src/plugins/intel_gpu/src/graph/eltwise.cpp]: Done

**Pull request merged**